models/app.models.js

```diff
@@ -1,5 +1,24 @@
 const db = require('../db/connection');
 
-exports.fetchTopics = () => {
-	return db.query('SELECT * FROM topics');
+exports.fetchTopics = async () => {
+	const results = await db.query('SELECT * FROM topics');
+
+	if (results.rowCount === 0) {
+		return Promise.reject({ code: 404 });
```

Comment:
I'm not sure there is any need for the Promise.reject here as we're not providing any criteria to the query